spec/features/user_can_delete_reviews_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+feature 'user can delete their own reviews', %{
+  As a user,
+  I want to delete my own reviews,
+  So that no one can see them
+} do
+
+  # Acceptance Criteria
+  # [√] I can delete my reviews from my profile page
+  # [√] If I delete a review, no one can see them anymore
+
+  let!(:user) { FactoryGirl.create(:user) }
+  let!(:author) { FactoryGirl.create(:author) }
+  let!(:genre) { FactoryGirl.create(:genre) }
+  let!(:book) do FactoryGirl.create(
+    :book,
+    user_id: user.id,
+    author_id: author.id,
+    genre_id: genre.id)
+  end
+  let!(:review) do FactoryGirl.create(
```

Comment:
Take a look at the comments on PR #17 regarding FactoryGirl associations and usage.